src/main/java/org/mondemand/RealTimeClient.java

```diff
@@ -0,0 +1,180 @@
+package org.mondemand;
+
+import java.net.InetAddress;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import org.apache.log4j.Logger;
+import org.mondemand.transport.LWESTransport;
+
+import com.google.common.util.concurrent.AtomicLongMap;
+
+/**
+ * This is an mondemand client that emit stats according to platformhash, site, aduit
+ *
+ */
+public class RealTimeClient {
+  private static final Logger LOG = Logger.getLogger(RealTimeClient.class);
+
+  protected final String host;
+  protected final int port;
+  protected final Client client;
+  protected final String clientName;
+  private ConcurrentMap<Context, AtomicLongMap<String>>  mondemandContent
+        = new ConcurrentHashMap<Context, AtomicLongMap<String>>();
+  private volatile boolean enabled = true;
+  protected final int INTERVAL = 60000;
+  protected boolean statsReset = true;
+
+  private static RealTimeClient rtClient = null;
```

Comment:
I would remove the singleton variable from this class.  It's not essential to this class, and if the user of this class wants to use a singleton instance of it, they can certainly use it in that way without this class having specific support for it.